java/jakarta/el/Util.java

```diff
@@ -380,11 +380,12 @@ static Method findMethod(Class<?> clazz, Object base, String methodName,
     private static final String paramString(Class<?>[] types) {
         if (types != null) {
             StringBuilder sb = new StringBuilder();
-            for (int i = 0; i < types.length; i++) {
-                if (types[i] == null) {
+            for (Class<?> type : types) {
+                if (type == null) {
                     sb.append("null, ");
-                } else {
-                    sb.append(types[i].getName()).append(", ");
+                }
+                else {
```

Comment:
These formatting changes need to be removed from this PR. Tomcat coding convention is to use `} else {` on a single line.

Reply:
The if-else formatting should be fixed now